CONTRIBUTING.md

```diff
@@ -51,10 +50,10 @@ maintaining quality documentation is one of our most important jobs. You can
 find more information about contributing to the documentation in the
 [docs/ directory](https://github.com/thepracticaldev/dev.to/blob/master/docs/contributing.md)!
 
-**Refactoring** code, or improving the code without modifying the behavior, is
+**Refactoring** code, or improving the code without modifying the behaviour, is
```

Comment:
:us: :eagle:

vs

:gb: :crown:

Reply:
Is this really worth changing? Shouldn't we aim for consistency here? I'm pretty sure we use US spellings almost everywhere, including the code base.

Reply:
This makes sense. @Computer-Man497, how do you feel about using the US spelling?